usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pytest
+
+
+@pytest.mark.skipinbeta
+def test_sles_hardened(host, get_release_value):
+    variant = get_release_value('VARIANT_ID') or ''
+
+    if "hardened" not in variant:
+        pytest.skip('Skipping non-hardened image')
```

Comment:
needs a return here or the tests following will be executed and non hardened images will fail testing